mineinabyss-features/src/main/kotlin/com/mineinabyss/curse/CurseAscensionListener.kt

```diff
@@ -54,6 +56,12 @@ class CurseAscensionListener : Listener {
             handleCurse(player, from, to)
     }
 
+    @EventHandler
```

Comment:
Id rather this also be in the other listener. Think about dependencies, curse has no need to depend on gsit so this listener shouldn't be here. Make the function inside here public and either make the listener a singleton `object` or put the handle curse function into a helper singleton.